kore/src/Kore/Repl/Data.hs

```diff
@@ -219,18 +220,20 @@ generalLogOptionsTransformer ::
     Log.KoreLogOptions ->
     Log.KoreLogOptions
 generalLogOptionsTransformer
-    logOptions@(GeneralLogOptions _ _ _ _)
```

Comment:
The purpose of this was to remind future authors about this function, where they have to copy each new field from `GeneralLogOptions` to `KoreLogOptions`. @ana-pantilie should I push a commit making `GeneralLogOptions` a field of `KoreLogOptions` instead of having each of them in both places?

Reply:
> making GeneralLogOptions a field of KoreLogOptions

Sorry, I don't understand this, why would this help?

Reply:
It would remove code duplication and the need to manually copy each field from `GeneralLogOptions` to `KoreLogOptions`

Reply:
That might make moving an existing field into the general log options (what was done in this PR) much harder, as now you have to adjust everywhere the field is used as well, right?

If the purpose of this line was just to generate a compiler error when `GeneralLogOptions` changes, then I can revert this change and add a comment about that.